src/components/atoms/productCategoryName.jsx

```diff
@@ -1,21 +1,6 @@
 import React from 'react';
 import { Text } from 'react-native';
 
-const ProductCategoryName = ({ name }) => {
-    return (
-        <Text
-            style={{
-                color: '#fff',
-                position: 'absolute',
-                bottom: 10,
-                fontSize: 20,
-                marginTop: 5,
-                textTransform: 'capitalize',
-            }}
-        >
-            {name}
-        </Text>
-    );
-};
+const ProductCategoryName = ({ name, style }) => <Text style={style}>{name}</Text>;
```

Comment:
Pas fan de l'écriture en une seule ligne
```suggestion
const ProductCategoryName = ({ name, style }) => (
    <Text style={style}>
        {name}
    </Text>
);
```

Reply:
hmmm bah c'est le prettier qui le raccourci comme ça, c'est soit ça soit obligé de faire avec un { return (....) } pour pas que le prettier le raccourcisse